src/components/Calendar/helpers/isDateWithinRange.js

```diff
@@ -0,0 +1,7 @@
+import compareDates from './compareDates';
+
+export default function isDateWithinRange(date, range) {
+    if (!date || !Array.isArray(range) || range.length <= 1) return false;
```

Comment:
I guess in the UCLV we learn to exit quickly in a function :) ... almost every code that I review of UCLV folks it's like that ... but actually it's difficult to read ... try it like this and check if makes sense for you

```js
if (data && Array.isArray(range) && range.length > 1) {
...
}
return false;
```

as a rule of thumbs try to avoid negations(!) when possible on if conditions